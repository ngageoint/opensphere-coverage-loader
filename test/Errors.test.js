const loader = require('../src');

describe('Errors', () => {
  test('Options Validation Error', () => {
    const err = () => loader.call({ query: { debug: 'foo' } });

    expect(err).toThrow();
    expect(err).toThrowErrorMatchingSnapshot();
  });
});
