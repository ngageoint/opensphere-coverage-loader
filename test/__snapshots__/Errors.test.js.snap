// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Options Validation Error 1`] = `
"Invalid options object. Istanbul Instrumenter Loader has been initialized using an options object that does not match the API schema.
 - options.debug should be a boolean."
`;
